monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpentelemetryReactiveIT.java

```diff
@@ -87,6 +89,35 @@ public void testSecurityEvents() {
 
     }
 
+    @Test
+    public void testExemplars() {
+        whenDoPingPongRequest();
+        whenDoPingPongRequest();
+        await().ignoreExceptions().atMost(30, TimeUnit.SECONDS)
+                .pollInterval(5, TimeUnit.SECONDS)
+                .untilAsserted(() -> {
+                    Response response = pingservice.given().get("/q/metrics");
+                    assertEquals(HttpStatus.SC_OK, response.statusCode());
+                    final String metricName = "http_server_requests_seconds_count";
+                    final String body = response.body().asString();
+                    String metric = Arrays.stream(body.split("\n"))
+                            .filter(line -> line.startsWith(metricName))
+                            .findFirst()
+                            .orElseThrow(() -> new AssertionError(metricName + " was not found in " + body));
+                    String[] content = metric.split(" ");
+                    assertEquals(6, content.length, "Some values are missing from " + metric);
+                    assertEquals("""
+                            http_server_requests_seconds_count{method="GET",outcome="SUCCESS",status="200",uri="/ping/pong"}
+                            """.strip(), content[0]);
+
+                    assertEquals("2.0", content[1], "Amount of events is wrong");
+                    assertEquals("#", content[2], "Unexpected exemplar separator!");
+                    assertTrue(content[3].contains("span_id"), "Exemplar doesn't contain span ID");
+                    assertTrue(content[3].contains("trace_id"), "Exemplar doesn't contain trace ID");
+                    assertEquals("1.0", content[4], "Unexpected exemplar value!");
```

Comment:
Based on this https://github.com/micrometer-metrics/micrometer/pull/3091 I expect that (at least) counter and timer are supported, but this only tests counter, correct?

Reply:
I was unable to find any documentation regarding meaning of values. Currently, this metrics looks like this:
`http_client_requests_seconds_count{clientName="localhost",method="GET",outcome="SUCCESS",status="200",uri="/hello"} 2.0 # {span_id="1f2d2ffc5464e4ed",trace_id="23385ada3cd0e5f14fd175a4955ceb6e"} 1.0 1740580104.884`. Reading from your comment it looks like both counter ("1.0") and timer ("174<...>" ) are there.